### PR TITLE
Reworks and Resprites Vampire

### PR DESCRIPTION
**Pull request by carlarctg**
# About the pull request

Added a variable to xeno_throw_human to disable the screenshake.

Headbite execution has been moved to its own ability. Sprite taken from TGMC.

Headbite is .25 seconds faster and heals 150 health instead of 100.

Removed Daze from Rush.

Flurry aims selected limb.

Tail jab has been changed from a sprite click ability into a telegraph ability like Scissor Cut or Pike.

Directly clicking a target with Tail jab buffs the jab to deal +15 damage and throw instead of step away. (Same distance)

Added XvX checks to all abilities.

Reduced Vampire speed bonus.

Resprited Vampire to be slightly more different from default Lurker, more similar to Ravager, and have a less intense palette.

# Explain why it's good for the game

> Headbite execution has been moved to (Wicked) Tail Stab, which has been readded to Vampire. Headbite execution does not need you to wait for the tail stab windup to finish to use on a target.

It was awkward sometimes to tail jab someone and fail and try for an execution instead. Tail Stab didn't really have a good reason to be removed from Vampire other than redundancy - letting it be the ability that executes instead will alleviate some of that redundancy. Since this would mean there's a 1-second windup in combat to utilize, I've made it so you don't need to wait for it.

> Headbite is .25 seconds faster and heals 150 health instead of 100.

Headbite right now is pretty much a useless flavor ability, 100 health regained after an engagement after a whole second of windup is simply not worth it for most practical purposes. This may help it be more useful in engagements.

> Removed Daze from Rush.

This served no real purpose other than pissing people off, the screenshake is already there to prevent PBs.

> Flurry aims selected limb.

I'm unsure on this one, but it will help with Vampries never managing to actually get any bonebreaks even after a whole moment stacking combos on marines.

> Tail jab has been changed from a sprite click ability into a telegraph ability like Scissor Cut or Pike.

You needed to be as precise as a Warrior, with the speed of a Runner, to use this ability. That's just too much precision - we want normal players to be able to use Vampire too, not just giga satan plabbliterators.

> Directly clicking a target with Tail jab buffs the jab to deal +15 damage and throw instead of step away. (Same distance)

That said, I liked how skill intensive the ability was, so I made it reward you for still aiming a direct sprite hit. The buffs aren't gamebreakingly good or anything, but they won't hurt, and it lets tail jab access throwing which has plenty of interactions.

> Added XvX checks to all abilities.

Tail jab could stun enemy queens or T3s and push them back, and the abilities did not have the XvX damage mult.

> Reduced Vampire speed bonus.

This made Vampire both annoying to fight and difficult to play due to its crack addict ridiculous speed. Halving it still makes it a fast strain without making it ridicuously quick.

> Resprited Vampire to be slightly more different from default Lurker, more similar to Ravager, and have a less intense palette.

The Vampire sprite is pretty bad, this has some admittedly too-minor changes that might just help differentiate it more from base Lurker than 'haha red'

# Testing Photographs and Procedure

<details>
<summary>Screenshots & Videos</summary>

Put screenshots and videos here with an empty line between the screenshots and the `<details>` tags.

</details>

# Changelog

:cl: Carlarc, Puckaboo
code: Added a variable to xeno_throw_human to disable the screenshake.
balance: Headbite execution has been moved to its own ability on the fourth slot. Ability sprites by Puckaboo!
balance: Headbite is .25 seconds faster and heals 150 health instead of 100.
balance: Removed Daze from Rush.
balance: Flurry aims selected limb.
balance: Tail jab has been changed from a sprite click ability into a telegraph ability like Scissor Cut or Pike.
balance: Directly clicking a target with Tail jab buffs the jab to deal +15 damage and throw instead of step away. (Same distance)
balance: Added XvX checks to all abilities.
balance: Reduced Vampire speed bonus.
balance: Resprited Vampire to be slightly more different from default Lurker, more similar to Ravager, and have a less intense palette.

/:cl:

### CODE REVIEW

**Comment by stanalbatross** [comment]
epic ci fail

**Comment by carlarctg** [comment]
I have no idea why my dme decided to explode

File: code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm [hunk 1]
```diff
@@ -172,7 +172,7 @@
 		T.update_xeno_hostile_hud()
 	to_chat(H, SPAN_XENOHIGHDANGER("You can move again!"))
 
-/proc/xeno_throw_human(mob/living/carbon/H, mob/living/carbon/Xenomorph/X, direction, distance)
+/proc/xeno_throw_human(mob/living/carbon/H, mob/living/carbon/Xenomorph/X, direction, distance, var/shake_camera = TRUE)
```
Comment (harryob) [inline code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm:175]: you don't need var here, the rest don't even have it, Consistency

**Review by harryob — APPROVED** [review]
i cant believe we don't have defines for "chest"

File: code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm (hunk 1 above)
Comment (carlarctg) [reply code/modules/mob/living/carbon/xenomorph/abilities/ability_helper_procs.dm:175]: youre a nerd

**Comment by carlarctg** [comment]
Updated
- Moved headbite from tail stab to its own ability
- Headbite sprite taken from TGMC
- Reduced throw distance of direct tail jab back to normal

**Review by nauticall — APPROVED** [review]
sprites look good! approved👍

**Review by morrowwolf — APPROVED** [review]
Some parts may need to be tuned down in the future but good for now.

**Pull request merged**